### PR TITLE
Added Japanese translation.

### PR DESCRIPTION
**Pull request by kzt206**
I translated to Japanese.
This project is very useful for LaTex beginners.

I am started git and github recently.
I'm not sure to pull request.

### CODE REVIEW

File: Translation:Japanese.md [hunk 1]
```diff
@@ -0,0 +1,503 @@
+English ·
```
Comment (i-ky) [inline Translation:Japanese.md]: This should link to English version:
```suggestion
[English](./readme.md) ·
```

File: Translation:Japanese.md [hunk 2]
```diff
@@ -0,0 +1,503 @@
...
+[中文](./Translation:Chinese.md) ·
+[Español](./Translation:Spanish.md) ·
+[Português](./Translation:Portuguese.md) ·
+[Français](./Translation:French.md) ·
+[Русский](./Translation:Russian.md) ·
+[Deutsch](./Translation:German.md) ·
+[日本語](.Translation:Japanese.md)
```
Comment (i-ky) [inline Translation:Japanese.md:8]: Add such link to your translation in other translations. But here remove the link:
```suggestion
日本語
```
Language without a link shows the reader which translation he/she is viewing. Have a look at other translations and do the same way.

File: Translation:Japanese.md [hunk 3]
```diff
@@ -0,0 +1,503 @@
...
+
+---
+
+# すぐに始める LaTeX 入門
+
+![](https://upload.wikimedia.org/wikipedia/commons/9/92/LaTeX_logo.svg)
+
+(Japanese translator's acknowledgement: I tried to translate this guide as the original meaning of the author as much as I could. If you think the translation is inappropriate, please feel free to modify it or add missing stuff you think would make this guide better. If you like this guide, please share to more people.)
+
+**初めに:** *この文書に書いていることは、私が大学での経験や目を通したいくつかの資料に基づくものです。私は、プロフェッショナルな仕事をしているわけでも、専門家でもなく、大いなる情熱を持つ学生です。問題セクションでは、誰でも議論に参加することを歓迎しますし、修正や加えるべきことがあれば、プルリクエストをすることができます。 もし、私のプロジェクトに価値があると思えば、ぜひ寄付[donation](#donation)をお願いします。*
```
Comment (i-ky) [inline Translation:Japanese.md]: You can translate text in square brackets (`[donation]`) as well.

**Review by i-ky — CHANGES_REQUESTED** [review]
I can't verify your Japanese text. Please Do the changes to the header and add links to your translation from other pages.

**Comment by i-ky** [comment]
Good job! Please add a link from `readme.md` and `Translation:*.md` files to your translation. See bc6e56ac398c47af34efc0ebd12d10ce79b23e90 as an example.

**Comment by luong-komorebi** [comment]
Awesome work, guys. I did send this to some of my friends who can read Japanese and they all commented LGTM <3

**Comment by luong-komorebi** [comment]
Thank you so much for your work @kzt206 and your reviews @i-ky

File: Translation:Japanese.md [hunk 4]
```diff
@@ -0,0 +1,502 @@
...
+## HOORAY!!
+
+:tada: 最後までこのガイドを読んでくれてありがとうございます。ここでは、LaTeXの基礎を学んだだけです。 :hammer:  
+もっとLaTeX のことを学びたければ、[ここ](http://www.latex-project.org/help/documentation/)を見たり、ウエブで調べてください。
+
+## ライセンス
+
+ [![](http://www.wtfpl.net/wp-content/uploads/2012/12/wtfpl-badge-1.png)](http://www.wtfpl.net/)
+
+**どうとでも勝手にしやがれクソッタレ・公衆利用許諾(契約)書**
```
Comment (luong-komorebi) [inline Translation:Japanese.md:492]: Oh boy, I am so curious how WTFPL is translated to Japanese.

**Pull request merged**